packages/labs/src/components/tag-input/_tag-input.scss

```diff
@@ -48,6 +48,10 @@ $ti-icon-padding-large: ($pt-input-height-large - $pt-icon-size-standard) / 2;
       padding: 0 ($input-padding-horizontal - $ti-padding);
     }
 
+    &[disabled] {
```

Comment:
nice, please also add support for `.pt-disabled` 👌 it's part of our generic modifiers